src/styles/components/buttons/styles.less

```diff
@@ -102,149 +107,132 @@ components/buttons.less
         &:last-child {
           margin-bottom: 0;
         }
-      }
-    }
-  }
 
-  /* -----------------------------------------------------------------------------
-  --------------------------------------------------------------------------------
+      }
 
-  Button Group
+    }
 
-  --------------------------------------------------------------------------------
-  ----------------------------------------------------------------------------- */
+  }
 
   .button-group {
-
     margin-bottom: @base-spacing-unit * 0.5;
 
     .dot {
-
       margin-right: @base-spacing-unit * 0.25;
-
     }
 
     .badge {
-
       margin-left: @base-spacing-unit * 0.25;
-
     }
 
     &-directions {
```

Comment:
Another one I'm concerned is a quick hack vs. fixing the problem in the CNVS button object directly.